lib/core/src/display/shape/text/text_field.rs

```diff
@@ -135,12 +135,35 @@ impl TextField {
     }
 
     /// Make change in text content.
+    ///
+    /// As an opposite to `edit` function, here we don't care about cursors, just do the change
+    /// described in `TextChange` structure.
     pub fn make_change(&mut self, change:TextChange, fonts:&mut FontRegistry) {
         self.content.make_change(change);
         self.assignment_update(fonts).update_after_text_edit();
         self.rendered.update_glyphs(&mut self.content,fonts);
     }
 
+    /// Get the selected text.
+    pub fn get_selected_text(&self) -> String {
+        let cursor_select  = |c:&Cursor| self.content.copy_fragment(c.selection_range());
+        let mut selections = self.cursors.cursors.iter().map(cursor_select);
+        selections.join("\n")
+    }
```

Comment:
What are you using it for? I'm asking because I dont think this is the correct API. If you have 3 cursors and you copy their selection and you paste it later still having 3 cursors, the text from first cursor should be pasted to the new first cursor etc (see intellij, vscode, and sublime for refernece)

Reply:
Yeah, but on the other hand when you have 3 cursors and make copy, and then paste it in an external editor you got 3 lines.

I can make pasting better, where in multiline clipboard and mutlicursor each line goes to another cursor.